### PR TITLE
{bio}[GCC/11.3.0,foss/2022a] MAKER v3.01.04, SNAP-HMM v20221022

### PR DESCRIPTION
**Pull request by smoors**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by smoors** [comment]
@boegelbot: please test @ generoso

**Comment by smoors** [comment]
@boegelbot please test @ jsc-zen2

File: easybuild/easyconfigs/m/MAKER/MAKER-3.01.04-foss-2022a.eb [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+        'source_urls': ['https://github.com/Yandell-Lab/maker/archive/refs/tags/'],
+        'start_dir': 'src',
+        'checksums': ['b9513585c90dc88288e903aff87bfddc7a66a39ec2b79619465ca7d679021fa7'],
+    }),
+]
+
+modextrapaths = {
+    'PERL5LIB': _libdir,
+    'ZOE': '$EBROOTSNAPMINHMM/Zoe',
+    'AUGUSTUS_CONFIG_PATH': '$EBROOTAUGUSTUS/config',
```
Comment (lexming) [inline easybuild/easyconfigs/m/MAKER/MAKER-3.01.04-foss-2022a.eb:91]: These are already by the respective dependencies, so it should not be needed, isn't it?

Comment (smoors) [reply easybuild/easyconfigs/m/MAKER/MAKER-3.01.04-foss-2022a.eb:91]: indeed, I'll remove them

**Comment by lexming** [comment]
@boegelbot: please test @ generoso

**Comment by lexming** [comment]
Going in, thanks @smoors !

**Review by lexming — APPROVED** [review]
LGTM

**Comment by lexming** [comment]
Going in, thanks @smoors !

**Pull request merged**